nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -1894,7 +1907,12 @@ void nxagentKeycodeConversionSetup(void)
                 "Activating KeyCode conversion.\n");
     #endif
 
-    nxagentKeycodeConversion = 1;
+    fprintf(stderr, "Info: Keycode conversion auto-determined as on\n");
+    nxagentKeycodeConversion = True;
```

Comment:
Need to check for `nxagentOption(KeycodeConversion) != KeycodeConversionOff` here, otherwise we'll ignore this parameter!

Reply:
no, this has already been checked at the beginning of the function.

Reply:
Indeed, I missed the `return` statements for some reason and assumed that these sections were fall-through...